### PR TITLE
ROX-9928: conditional violation name

### PR DESCRIPTION
**Pull request by jschnath**
## Description

As presented to the product team with no objections, simply removing the product name out of the json file is much simpler than finding a complex solution for templating.

## Checklist
- [ ] ~Investigated and inspected CI test results~
- [ ] ~Unit test and regression tests added~
- [x] Evaluated and added CHANGELOG entry if required
- [ ] ~Determined and documented upgrade steps~
- [ ] ~Documented user facing changes (create PR based on [stackrox/openshift-docs](https://github.com/stackrox/openshift-docs) and merge into [rhacs-docs](https://github.com/openshift/openshift-docs/tree/rhacs-docs))~

### CODE REVIEW

File: migrator/migrations/m_98_to_99_violation_report_branding/migration.go [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+		},
+	}
+
+	//go:embed policies_before_and_after
+	policyDiffFS embed.FS
+
+	// We will want to migrate only if the existing policy sections and title haven't changed.
+	fieldsToCompare = []policymigrationhelper.FieldComparator{
+		policymigrationhelper.PolicySectionComparator,
+		policymigrationhelper.NameComparator,
```
Comment (theencee) [inline migrator/migrations/m_98_to_99_violation_report_branding/migration.go:33]: Also add `policymigrationhelper.DescriptionComparator`. This migration is changing the description as well and we don't want to change it if the user has changed it.

**Review by theencee — COMMENTED** [review]
Please add:

- Changes to migrator/runner/all.go to include the new migration
- Increase version to 99 in pkg/migrations/internal/seq_num.go
- Change log to CHANGELOG.md  to mention the fact that policy is being renamed.

**Review by fredrb — APPROVED** [review]
Looks good to me. Just please check if that upgrade test is failing because of the migration.

**Comment by jschnath** [comment]
No, looks like it's failing when polling for open source images. Definitely something we should look into but unrelated to this migration

**Pull request merged**